lib/plugin/selenoid.js

```diff
@@ -0,0 +1,155 @@
+const util = require('util');
+const axios = require('axios').default;
+const exec = util.promisify(require('child_process').exec);
+const {
+  container, event, recorder, output,
+} = require('../index');
+
+let dockerCreateScript = `docker create --rm --name $name$ -p 4444:4444 -v /var/run/docker.sock:/var/run/docker.sock -v ${global.codecept_dir}/:/etc/selenoid/:ro -v ${global.output_dir}/video/:/opt/selenoid/video/ -v ${global.output_dir}/logs/:/opt/selenoid/logs/ -e OVERRIDE_VIDEO_OUTPUT_DIR=${global.output_dir}/video/ $additionalParams$ aerokube/selenoid:latest-release -log-output-dir /opt/selenoid/logs`;
```

Comment:
it would be nice if you could compose the string the way it is split by multiple lines here
maybe use it as array of arguments.
At this moment it is hard to read and understand which actual parameters are passed in

Reply:
Port should be configured as well

Reply:
@DavertMik sure, Got it